docs/tasks/fact-check.md

```diff
@@ -24,4 +24,23 @@ environment variable in the [`app.json`](../../app.json), for example:
 `"RATES_QUERY_DATE": "2020-04-01"` - don't forget to remove this before
```

Comment:
while you're here, could you perhaps add how to get the login creds for the review apps frontend as well?